nslsii/__init__.py

```diff
@@ -160,14 +160,29 @@ def configure_base(user_ns, broker_name, *,
         from ophyd import setup_ophyd
         setup_ophyd()
 
-    if not ophyd_logging:
-        # Turn on error-level logging, particularly useful for knowing when
-        # pyepics callbacks fail.
+    if configure_logging:
         import logging
-        import ophyd.ophydobj
-        ch = logging.StreamHandler()
-        ch.setLevel(logging.ERROR)
-        ophyd.ophydobj.logger.addHandler(ch)
+        from logging.handlers import TimedRotatingFileHandler
+
+        log_file_handler = TimedRotatingFileHandler(
+            filename=os.path.expanduser("/var/log/bluesky/bluesky.log"),
+            when="W0",
+            backupCount=10
+        )
+        log_file_handler.setLevel("INFO")
+
+        logging.getLogger("bluesky").addHandler(log_file_handler)
+        logging.getLogger("bluesky.*").addHandler(log_file_handler)
```

Comment:
I am 90% sure that a handler attached to `bluesky` is also effectively attached to `bluesky.*`, making these second lines redundant.

Reply:
I checked and you are correct.